src/app.ts

```diff
@@ -38,7 +38,9 @@ declare module "express-session" {
 }
 
 app.get('/', getHomePage);
+
 app.get('/job-roles', getAllJobRoles);
+app.get('/job-roles-:id', getJobRole)
```

Comment:
```suggestion
app.get('/job-roles/:id', getJobRole)
```

Reply:
We had it like this because the CSS file couldn't be referenced with the /